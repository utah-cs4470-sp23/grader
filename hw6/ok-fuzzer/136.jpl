show [[656, ((- (if true then 88 else 806)) * 177)], (if (if ((- 630) != (19 % 226)) then (54.78 > (if true then 20.87 else 45.21)) else false) then [(if (150 >= 432) then 134 else 859)] else (if ((26.62 / 33.05) <= 2.61) then [(599 - 528), (103 * 968), 685] else [(51 - 755), (- 169), (522 * 291)]))]

