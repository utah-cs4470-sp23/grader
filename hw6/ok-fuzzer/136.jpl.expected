(ShowCmd
 (ArrayLiteralExpr
  (ArrayType (ArrayType (IntType) 1) 1)
  (ArrayLiteralExpr
   (ArrayType (IntType) 1)
   (IntExpr (IntType) 656)
   (BinopExpr
    (IntType)
    (UnopExpr
     (IntType)
     -
     (IfExpr
      (IntType)
      (TrueExpr)
      (IntExpr (IntType) 88)
      (IntExpr (IntType) 806)))
    *
    (IntExpr (IntType) 177)))
  (IfExpr
   (ArrayType (IntType) 1)
   (IfExpr
    (BoolType)
    (BinopExpr
     (BoolType)
     (UnopExpr (IntType) - (IntExpr (IntType) 630))
     !=
     (BinopExpr (IntType) (IntExpr (IntType) 19) % (IntExpr (IntType) 226)))
    (BinopExpr
     (BoolType)
     (FloatExpr (FloatType) 54)
     >
     (IfExpr
      (FloatType)
      (TrueExpr)
      (FloatExpr (FloatType) 20)
      (FloatExpr (FloatType) 45)))
    (FalseExpr))
   (ArrayLiteralExpr
    (ArrayType (IntType) 1)
    (IfExpr
     (IntType)
     (BinopExpr (BoolType) (IntExpr (IntType) 150) >= (IntExpr (IntType) 432))
     (IntExpr (IntType) 134)
     (IntExpr (IntType) 859)))
   (IfExpr
    (ArrayType (IntType) 1)
    (BinopExpr
     (BoolType)
     (BinopExpr
      (FloatType)
      (FloatExpr (FloatType) 26)
      /
      (FloatExpr (FloatType) 33))
     <=
     (FloatExpr (FloatType) 2))
    (ArrayLiteralExpr
     (ArrayType (IntType) 1)
     (BinopExpr (IntType) (IntExpr (IntType) 599) - (IntExpr (IntType) 528))
     (BinopExpr (IntType) (IntExpr (IntType) 103) * (IntExpr (IntType) 968))
     (IntExpr (IntType) 685))
    (ArrayLiteralExpr
     (ArrayType (IntType) 1)
     (BinopExpr (IntType) (IntExpr (IntType) 51) - (IntExpr (IntType) 755))
     (UnopExpr (IntType) - (IntExpr (IntType) 169))
     (BinopExpr (IntType) (IntExpr (IntType) 522) * (IntExpr (IntType) 291)))))))
